### ISSUE TITLE
Undefined argument "qtype" in CSW update transaction

### ISSUE THREAD

**Issue by kdeininger** [geopython/OWSLib#354]
Trying to use the CSW update transaction, I get the following exception:

`NameError: global name 'qtype' is not defined`

It points to [csw.py, line 448](https://github.com/geopython/OWSLib/blob/16fc43a8b90a34c1075ce1f08b2585ff40e82fb8/owslib/csw.py#L448), where the second argument "qtype" is an undefined variable. Maybe it has to be set to `None`, as for the delete transaction below, or a default value should be assigned before.

**Comment by tomkralidis** [comment]
Can you provide a code snippet so I can reproduce? As well as version of OWSLib.

**Comment by kdeininger** [comment]
I use the latest release 0.13.0. Here's a simple code snippet:

```python
csw = CatalogueServiceWeb(
    'https://www.geocat.ch/geonetwork/srv/ger/csw-publication',
    username='********',
    password='********',
    skip_caps=True
)

for item in items:
    csw.transaction(
        ttype='update',
        typename='gmd:MD_Metadata',
        propertyname='/'.join([
            'gmd:identificationInfo',
            'che:CHE_MD_DataIdentification',
            'gmd:citation',
            'gmd:CI_Citation',
            'gmd:date',
            'gmd:CI_Date',
            'gmd:date',
            'gco:Date'
        ]),
        propertyvalue=item.get('date'),
        identifier=item.get('uuid')
    )
```

Everything works fine, if I replace "qtype" with "None" at the mentioned line in csw.py.

**Comment by tomkralidis** [comment]
@kdeininger fixed in master in e185a50028dc566881005a2ad2d750dec00079a5.  Can you confirm and close the issue if this works for you?

**Comment by kdeininger** [comment]
Works perfect now. Thanks a lot!
